src/views/AuthorityView/AuthorityView.js

```diff
@@ -89,38 +112,47 @@ const AuthorityView = ({
     return stripes.hasPerm('ui-marc-authorities.authority-record.edit');
   };
 
+  const hasDeletePermission = () => {
+    return stripes.hasPerm('ui-marc-authorities.authority-record.delete');
+  };
+
   const markHighlightedFields = () => {
     const highlightAuthRefFields = {
-      'Authorized': /1\d\d/,
-      'Reference': /4\d\d/,
+      Authorized: /1\d\d/,
+      Reference: /4\d\d/,
       'Auth/Ref': /5\d\d/,
     };
 
-    const marcFields = marcSource.data.parsedRecord.content.fields.map(field => {
-      const tag = Object.keys(field)[0];
+    const marcFields = marcSource.data.parsedRecord.content.fields.map(
```

Comment:
There are a lot of changes of formatting. Do you have some plugin that does it automatically?